README.md

```diff
@@ -0,0 +1,38 @@
+arrayfire-dotnet
+==============
+
+This repository contains the files required to use ArrayFire from any .net Language such as C# or F#
+
+Prerequisites
+---------------
+
+- The latest version of ArrayFire. You can get ArrayFire using one of the following:
+    - [Download binary installers](http://www.arrayfire.com/download)
+    - [Build from source](https://github.com/arrayfire/arrayfire)
+
+- Visual Studio 2012 or above, 2015 recommended
+    - [Download Visual Studio 2015 Community Free](https://www.visualstudio.com/en-us/downloads/download-visual-studio-vs.aspx)
```

Comment:
Can we point to the VS2013 link as well. We base ArrayFire on VS2013, so having that link here would be great.
Plus, CUDA does not have support for VS2015 yet.